cms_notification/models/notification_panel_form.py

```diff
@@ -1,3 +1,4 @@
+        # no description needed, at least for now :)
```

Comment:
what's this? 😃

Reply:
https://github.com/OCA/website-cms/blob/11.0/cms_form/templates/form.xml#L17

Reply:
WTFFFFFF that's really sad copy/paste error of my editor :(
thanks for catchin it! :smile: